### PR TITLE
started

### PR DESCRIPTION
**Pull request by dbvu88**

### CODE REVIEW

**Review by NickStrick — CHANGES_REQUESTED** [review]
make sure you give me more commits 😠 these make easier to looks at and review your code, and also makes your github look better.
these concepts are important to everything were doing this month so i encourage you to go back and complete this, but current lessons and projects take priority over missed projects.

File: Project-II-Calculator/calculator/src/App.js [hunk 1]
```diff
@@ -1,21 +1,91 @@
...
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+
+
+// calculator's logic
+function buttonLister(e) {
+  console.log(e.toElement.innerText);
+  console.log(e.toElement.classList[1]);
+}
+
+function onClickHandler(e) {
```
Comment (NickStrick) [inline Project-II-Calculator/calculator/src/App.js:14]: wow awesome looking stretch great job on getting these handlers started.

File: Project-II-Calculator/calculator/src/App.js [hunk 2]
```diff
@@ -1,21 +1,91 @@
...
+        display.innerText = "";
+        break;
+      }
+      default: display.innerText += value;
+    }
+  }
+}
+
+// app start
+document.addEventListener('click', (e) => onClickHandler(e) );
```
Comment (NickStrick) [inline Project-II-Calculator/calculator/src/App.js:45]: so in react we can add an onClick property to our html tags and just pass the function into them inside our child components that your returning in your .map function.

i encourage you to take a look at that:
https://reactjs.org/docs/handling-events.html

**Review by NickStrick — APPROVED** [review]
looks great! great use of props and components, demonstrates a good understanding of the material so far, but dont allow that to be an excuse to be lazy, repetition is the key to a strong foundation in learning new concepts

**Comment by dbvu88** [comment]
> looks great! great use of props and components, demonstrates a good understanding of the material so far, but dont allow that to be an excuse to be lazy, repetition is the key to a strong foundation in learning new concepts



> looks great! great use of props and components, demonstrates a good understanding of the material so far, but dont allow that to be an excuse to be lazy, repetition is the key to a strong foundation in learning new concepts

Thanks for the feedback! By the way, I'm not being lazy, I just have a lot to prioritize, so I got worked up by the end of the day, and lost focus a little. But, I always have a plan to get back on track. I just don't communicate that relly well.

**Pull request merged**